client/components/jetpack/upsell-product-card/test/index.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * @jest-environment jsdom
+ */
+import { PRODUCT_JETPACK_SCAN } from '@automattic/calypso-products';
+import { FEATURE_TYPE_JETPACK_SCAN } from '@automattic/calypso-products/src/constants/features';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import UpsellProductCard from 'calypso/components/jetpack/upsell-product-card';
+
+jest.mock( 'calypso/state/ui/selectors/get-selected-site-id', () => jest.fn() );
+
+jest.mock( 'calypso/my-sites/plans/jetpack-plans/use-item-price', () =>
+	jest.fn().mockReturnValue( {
```

Comment:
Note that we now have two additional keys returned:

`originalPriceTotal`
`discountedPriceTotal`